Tomlyn.Extensions.Configuration/TomlConfigurationFileParser.cs

```diff
@@ -69,16 +68,13 @@ private void VisitObject(TomlObject obj)
                 case TomlArray tomlArray:
                     VisitArray(tomlArray);
                     break;
-                case TomlValue tomlValue:
-                    var key = _paths.Peek();
-                    if (_data.ContainsKey(key))
-                    {
-                        throw new FormatException($"Key {key} is duplicated");
```

Comment:
I think the exception here was needed otherwise user might not be aware about configuration keys case insensitivity